### PR TITLE
Auxiliary Atom Integration

### PR DESCRIPTION
**Pull request by LATaylor-guardian**

### CODE REVIEW

File: restaurants/src/test/scala/com/gu/restaurant_review_parser/parsers/RestaurantReviewProcessorSpec.scala [hunk 1]
```diff
@@ -60,6 +61,7 @@ class RestaurantReviewProcessorSpec extends FunSuite with Matchers {
       review.approximateLocation.get shouldBe ApproximateLocation("London N1")
       review.reviewSnippet shouldBe Some(ReviewSnippet("As everyone in the room applies tongues to bricks, all I think is, someone's having a laugh"))
       review.reviewer shouldBe "Marina O'Loughlin"
+      review.internalComposerCode shouldBe Some("this-is-a-composer-code")
```
Comment (mchv) [inline restaurants/src/test/scala/com/gu/restaurant_review_parser/parsers/RestaurantReviewProcessorSpec.scala]: could we use an `md5` as a real internal composer code

Comment (LATaylor-guardian) [inline restaurants/src/test/scala/com/gu/restaurant_review_parser/parsers/RestaurantReviewProcessorSpec.scala]: ala `5829bef2f7d050b15a62881c`?

**Comment by mchv** [comment]
👍

**Comment by tomrf1** [comment]
👍

**Pull request merged**